frontend/src/metabase/env.ts

```diff
@@ -13,3 +13,6 @@ export const isChartsDebugLoggingEnabled =
   process.env.MB_LOG_CHARTS_DEBUG === "true";
 
 export const isEmbeddingSdk = !!process.env.IS_EMBEDDING_SDK;
+
+export const isBugReportingEnabled =
+  process.env.MB_BUG_REPORTING_ENABLED === "true";
```

Comment:
This would require us to a build a separate bundle to enable this - I think instead we should use an env var on the server to turn it on